dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -190,7 +202,7 @@ def get_default_platform_machine() -> str:
 CURRENT_PYTHON_MAJOR_MINOR_VERSIONS = ALL_PYTHON_MAJOR_MINOR_VERSIONS
 CURRENT_POSTGRES_VERSIONS = ["11", "12", "13", "14", "15"]
 DEFAULT_POSTGRES_VERSION = CURRENT_POSTGRES_VERSIONS[0]
-CURRENT_MYSQL_VERSIONS = ["5.7", "8"]
+CURRENT_MYSQL_VERSIONS = [*MYSQL_OLD_RELEASES, *MYSQL_LTS_RELEASES]
```

Comment:
Hmm. Don't we want to test Airflow in CI for the INNOVATION releases as well? I think we should - which means that CURRENT_MYSQL_VERSIONS should also contain `MYSQL_INNOVATION_RELEASE`.

Or maybe there is a good reason why we should not include it @Taragolis ?

Reply:
As far as I understand 8.1 it is also GA release, but with short live-cycle, I've just keep long-term supported releases when create PR.

Good Reasons To include:
- We might have more info about changes in the future LTS, so potentially we would spend less time to include new LTS release

Potential disadvantages:
- We need to bump this version every 3 months (approx)
- Something might goes wrong, but in this case we could disable it until resolve

Reply:
Yeah. Let's try.. We can always disable it in CI, but I am big fan of finding (and solving) problems as early as possible, that has the advantage of being able to react incrementally and fix the problems as they appear - I am sure soon we will have users who will start using 8.1 (It's **just** been released in July). I prefer to find out we have problems from our CI rather than from our users who will complain and demand to get it fixed "ASAP".

Reply:
I guess we run for about 1 month against MySQL 8.1 😄 
So on this case we pretty safe